README.md

```diff
@@ -1,2 +1,285 @@
-# panther_analysis_tool
-Command line tool for working with Panther rules and policies
+<h1 align="center">Panther Analysis Tool</h1>
+
+<p align="center">
+  <i>Command Line Panther Analysis Management</i>
+</p>
+
+<p align="center">
+  <a href="https://docs.runpanther.io">Documentation</a> |
+  <a href="https://docs.runpanther.io/quick-start">Quick Start</a>
+</p>
+
+<p align="center">
+  <a href="https://gitter.im/runpanther/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge"><img src="https://badges.gitter.im/runpanther/community.svg" alt="Gitter"/></a>
+  <a href="https://circleci.com/gh/panther-labs/panther-analysis"><img src="https://circleci.com/gh/panther-labs/panther-analysis.svg?style=svg" alt="CircleCI"/></a>
+  <a href="https://cla-assistant.io/panther-labs/panther-analysis" alt="CLA Assistant"><img src="https://cla-assistant.io/readme/badge/panther-labs/panther-analysis"/></a>
```

Comment:
I'm not sure if we have CLA enabled for this repo yet, but if so change the link

Reply:
I don't think so either, removing it for now.